### PR TITLE
feat: integrate pallet-elections-phragmen for general council voting

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

File: parachain/runtime/src/lib.rs [hunk 1]
```diff
@@ -367,6 +367,42 @@ impl pallet_treasury::Config for Runtime {
     type MaxApprovals = MaxApprovals;
 }
 
+pub const fn deposit(items: u32, bytes: u32) -> Balance {
+    items as Balance * KINT.one() + (bytes as Balance) * KINT.one()
```
Comment (sander2) [inline parachain/runtime/src/lib.rs:371]: I don't know the `pallet_elections_phragmen` pallet, but this looks weird. VotingBondBase is apparently 65 kint, while VotingBondFactor is 32 kint? Some additional comment on what this is exactly would be very welcome

Comment (gregdhill) [reply parachain/runtime/src/lib.rs]: Copied the exact setup from Kusama's runtime (since we use the same units) and left additional comments.

Comment (sander2) [reply parachain/runtime/src/lib.rs]: Ah your new version pf the deposit function looks at least plausible now🙂 guess we'll trust kusama for now

Comment (sander2) [reply parachain/runtime/src/lib.rs]: Ah your new version pf the deposit function looks at least plausible now🙂

**Review by sander2 — APPROVED** [review]

**Pull request merged**